CHANGELOG.md

```diff
@@ -1,8 +1,51 @@
 ## 1.0.0-beta+3
 
-- **BREAKING CHANGE**: Throws in bootstrapping if the root component does not
-  use default change detection. AngularDart does not support `OnPush` or other
-  change detection strategies on the root component.
+### Breaking Changes & Deprecations
+
+- Throws in bootstrapping if the root component does not use default change
+  detection. AngularDart does not support `OnPush` or other change detection
+  strategies on the root component.
+  
+- Pub serve now defaults to a random unused port (instead of `8080`) and
+  `--port` is deprecated. Going forward the supported way to supply this
+  argument is via `--serve-arg`:
+
+```bash
+$ pub run angular_test --serve-arg=port=1234
+```
+
+- Option `--run-test-flag` (`-t`) is now **deprecated**, and no longer has a
+  default value of `aot`. Tags are still highly encouraged in order to have
+  faster compilation times! Use `--test-arg` instead:
+
+```bash
+$ pub run angular_test --test-arg=--tags=aot
+```
+
+- Option `--platform` (`-p`) is now **Deprecated**, and no longer has a default
+  value of `content-shell`, which was not always installed on host machines.
+  Instead use `--test-arg`:
+
+```bash
+$ pub run angular_test --test-arg=--platform=content-shell
+```
+
+- Option `--name` (`-n`) and `--simple-name` (`-N`) are also deprecated. Use
+  `--test-arg=--name=` and `--test-arg=--simple-name=` instead.
```

Comment:
Is there documentation for this besides the changelog and the sample flags in the README? It wasn't obvious at first that you can provide multiple --test-arg's